test/test.js

```diff
@@ -19,45 +33,60 @@ describe('GET /hello', () => {
 });
 
 describe('Invoking searchRepositories', () => {
+    const mockResponse = {
+        createdAt: '2020',
+        collaborators: {
+            totalCount: 1,
+            edges: [{ node: { login: 'bela' } }]
+        }
+    };
+
     it('should return dummy response', async () => {
-        githubAPIMock.post('/graphql').reply(200,{
-            data: {
-                createdAt: '2020',
-                collaborators: {
-                    totalCount: 1,
-                    edges: [{
-                        node: {
-                            login: 'bela'
-                        }
-                    }
-                    ]
-                }
-            }
+
+        githubAPIMock.post('/graphql').reply(200, { data: mockResponse });
+
+        const response = await github.searchRepositories('WordsMemorizer');
+        response.should.deep.equal(mockResponse);
+    });
+
+    it('should include "$queryString" query variable', async () => {
+        githubAPIMock.post('/graphql').reply(200, function (uri, requestBody) {
+            requestBody.query.should.to.match(/\$queryString:String!/g, '"queryString" is missing!');
+            requestBody.query.should.to.match(/query:\s*\$queryString/g, '"queryString" is missing!');
```

Comment:
You can look for an exact match for the whole query string generated here: https://github.com/RisingStack/bootcamp-roland/blob/Model-for-GitHub-API/services/github.js#L19

Reply:
I am unable to do an exact match for the hole query string.